gitlab/resource_gitlab_group_label.go

```diff
@@ -81,8 +83,7 @@ func resourceGitlabGroupLabelRead(d *schema.ResourceData, meta interface{}) erro
 		}
 	}
 	if !found {
-		log.Printf("[WARN] removing label %s from state because it no longer exists in gitlab", labelName)
-		d.SetId("")
+		return errors.New(fmt.Sprintf("label %s no longer exists in group %s", labelName, group))
```

Comment:
I would say can not be found.
It does not exist or the user does not have the right to see it.

Reply:
reworded the message